src/OpenTelemetry.Instrumentation.Runtime/RuntimeMetrics.cs

```diff
@@ -74,7 +74,7 @@ static RuntimeMetrics()
                     return new Measurement<long>[] { new(GC.GetGCMemoryInfo().TotalCommittedBytes) };
                 },
                 unit: "bytes",
-                description: "The amount of committed virtual memory for the managed GC heap, as observed during the latest garbage collection. Committed virtual memory may be larger than the heap size because it includes both memory for storing existing objects (the heap size) and some extra memory that is ready to handle newly allocated objects in the future. The value will be unavailable until garbage collection has occurred.");
+                description: "The amount of committed virtual memory for the managed GC heap, as observed during the latest garbage collection. Committed virtual memory may be larger than the heap size because it includes both memory for storing existing objects (the heap size) and some extra memory that is ready to handle newly allocated objects in the future. The value will be unavailable until atleast one garbage collection has occurred.");
```

Comment:
```suggestion
                description: "The amount of committed virtual memory for the managed GC heap, as observed during the latest garbage collection. Committed virtual memory may be larger than the heap size because it includes both memory for storing existing objects (the heap size) and some extra memory that is ready to handle newly allocated objects in the future. The value will be unavailable until at least one garbage collection has occurred.");
```

Reply:
> Maybe update the README.md in the same PR?

Will result in conflicts. Before release, a manual review is needed to make sure both are in sync. Updates to readme.md can occur anytime, but changes to instrument name, description could be considered breaking change.